packages/compass-e2e-tests/helpers/commands/connect-with-connection-string.ts

```diff
@@ -8,13 +12,38 @@ export async function connectWithConnectionString(
   connectionStatus: 'success' | 'failure' | 'either' = 'success',
   timeout?: number
 ): Promise<void> {
-  const sidebar = await browser.$(Selectors.Sidebar);
-  if (await sidebar.isDisplayed()) {
-    await browser.disconnect();
+  await browser.disconnect();
+
+  if (TEST_MULTIPLE_CONNECTIONS) {
```

Comment:
This is probably going to be changed once we add tests for multiple-connection-specific paths. But we can only address a connection by name so if two or more of them have the same name then the css selector's gonna match multiple elements and nothing will work. This is especially a problem because the tests keep connecting with the form again and again with the same localhost:27091 connection string, adding another connection every time.